simulation/simtypes/randutil.go

```diff
@@ -74,3 +76,17 @@ func (sim *SimCtx) RandSubsetCoins(coins sdk.Coins) sdk.Coins {
 	r := sim.GetSeededRand("random subset coins")
 	return sdkrand.RandSubsetCoins(r, coins)
 }
+
+// Random float point from 0-1
+func (sim *SimCtx) RandomWeight(maxVal sdk.Dec) (sdk.Dec, error) {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Let's not use time.Now(). We should be able to replay all randomness from a daterministiv seed

Reply:
removed to use RandomDecAmount